diff --git a/data/qalculate-gtk.appdata.xml.in b/data/qalculate-gtk.appdata.xml.in
index 4c2e95b..2085e45 100644
--- a/data/qalculate-gtk.appdata.xml.in
+++ b/data/qalculate-gtk.appdata.xml.in
@@ -43,9 +43,6 @@
   <kudo>UserDocs</kudo>
  </kudos>
  <launchable type="desktop-id">qalculate-gtk.desktop</launchable>
- <provides>
-  <binary>qalculate-gtk</binary>
- </provides>
  <translation type="gettext">qalculate-gtk</translation>
  <releases>
   <release version="3.11.0" date="2020-06-22">
 
