diff --git a/configure.ac b/configure.ac
index be723fe..6cedf0f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -87,7 +87,7 @@ GTK_LIBS="-export-dynamic $GTK_LIBS"
 
 AC_ARG_WITH([webkitgtk], AS_HELP_STRING([--with-webkitgtk], [use WebKitGTK for display of the user manual]))
 AS_IF([test "x$with_webkitgtk" = "xyes"], [
-	PKG_CHECK_MODULES([WEBKITGTK], [webkit2gtk-4.0])
+	PKG_CHECK_MODULES([WEBKITGTK], [webkit2gtk-4.1])
 	AC_DEFINE([USE_WEBKITGTK], [1], [Use WebKitGTK])
 	AC_SUBST(WEBKITGTK_CFLAGS)
 	AC_SUBST(WEBKITGTK_LIBS)
--- a/configure
+++ b/configure
@@ -18030,12 +18030,12 @@
     pkg_cv_WEBKITGTK_CFLAGS="$WEBKITGTK_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"webkit2gtk-4.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "webkit2gtk-4.0") 2>&5
+    { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"webkit2gtk-4.1\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "webkit2gtk-4.1") 2>&5
   ac_status=$?
   printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_WEBKITGTK_CFLAGS=`$PKG_CONFIG --cflags "webkit2gtk-4.0" 2>/dev/null`
+  pkg_cv_WEBKITGTK_CFLAGS=`$PKG_CONFIG --cflags "webkit2gtk-4.1" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -18047,12 +18047,12 @@
     pkg_cv_WEBKITGTK_LIBS="$WEBKITGTK_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"webkit2gtk-4.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "webkit2gtk-4.0") 2>&5
+    { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"webkit2gtk-4.1\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "webkit2gtk-4.1") 2>&5
   ac_status=$?
   printf "%s\n" "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_WEBKITGTK_LIBS=`$PKG_CONFIG --libs "webkit2gtk-4.0" 2>/dev/null`
+  pkg_cv_WEBKITGTK_LIBS=`$PKG_CONFIG --libs "webkit2gtk-4.1" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -18073,14 +18073,14 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        WEBKITGTK_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "webkit2gtk-4.0" 2>&1`
+	        WEBKITGTK_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "webkit2gtk-4.1" 2>&1`
         else
-	        WEBKITGTK_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "webkit2gtk-4.0" 2>&1`
+	        WEBKITGTK_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "webkit2gtk-4.1" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$WEBKITGTK_PKG_ERRORS" >&5
 
-	as_fn_error $? "Package requirements (webkit2gtk-4.0) were not met:
+	as_fn_error $? "Package requirements (webkit2gtk-4.1) were not met:
 
 $WEBKITGTK_PKG_ERRORS
 
